react/week3/todo-app/src/Components/Todo.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import Border from "./Border";
+import PropTypes from "prop-types";
+function Todo({ toDo, deleteTodo, updateTodo }) {
+  const [status, setStatus] = useState(false);
+  const [isEditing, setEditState] = useState(false);
+  const [newdescription, setDescription] = useState(toDo.description);
```

Comment:
```suggestion
  const [newDescription, setNewDescription] = useState(toDo.description);
```
https://reactjs.org/docs/hooks-state.html